src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -257,13 +258,17 @@ private static void initLogLevel(Config config) {
             Platform platformInstance;
             switch (platform.type) {
                 case "mturk":
+                    MturkPlatform mturkPlatform;
                     //FIXME remove the sandbox url - but I am to paranoid
-                    platformInstance = new MturkPlatform(platform.user,
+                    platformInstance = mturkPlatform = new MturkPlatform(platform.user,
```

Comment:
I added the feature tonight to the jdk and sneaked the new binary with my hexxor skills on your laptop. Better protect yourself. 
tldr; yes.